cluster/environments/azure-multiple-clusters/aks-variables.tf

```diff
@@ -42,11 +41,16 @@ variable "gitops_ssh_key" {
 
 variable "aks_client_role_assignment_role" {
   type    = "string"
-  default = "Contributor"
+  default = "Owner"
```

Comment:
Why is owner needed here?

Reply:
AKS Service Principal do not need `Owner` role to modify networking resources. The `Contributor` role on a resource group where networking resources exists.